.bandit

```diff
@@ -0,0 +1,10 @@
+[bandit]
```

Comment:
can this live in the `setup.cfg` file instead?

Reply:
Great idea! Yes, it works